### PR TITLE
Feature/5678 automate test

### PR DESCRIPTION
**Pull request by infojunkie**

### CODE REVIEW

File: test/spec/app_spec_helpers.rb [hunk 1]
```diff
@@ -44,10 +44,11 @@ def twitter_login
       @driver.page_source.include?("#{@config['twitter_name']}")
     }
   end
-
+	#<button id="twitter-login" class="login-menu__button login-menu__button--twitter"><span>Sign in with Twitter</span></button>
```
Comment (caiosba) [inline test/spec/app_spec_helpers.rb]: Please remove this line

**Comment by infojunkie** [comment]
- I think the CSS changes are from merging `develop` back into this branch. I am currently checking this.
- @clarissacastella, for indentation, plz make sure your editor uses 2 spaces, not tabs.
- @caiosba: how about we run the quick test ONLY on Travis?

File: test/spec/app_spec_helpers.rb (hunk 1 above)
Comment (clarissacastella) [inline test/spec/app_spec_helpers.rb]: Sure. Done.

**Comment by caiosba** [comment]
> how about we run the quick test ONLY on Travis?

I feel more confident if we run both :)

**Pull request closed without merging**